### PR TITLE
Add first letter as capital and ..

### PR DESCRIPTION
**Pull request by gitgregor**
It is ok for now

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>project docs</title>
+    <title>Project documents</title>
```
Comment (gitgregor) [inline index.html:6]: looks good

**Review by gitgregor — COMMENTED** [review]
everything all right

**Comment by Marcytest** [comment]
Then I merge it with master now

**Comment by gitgregor** [comment]
ok Marcy get merge

**Pull request merged**